aggregate/PUMS/pums_0812_1519_economics.py

```diff
@@ -0,0 +1,228 @@
+import pandas as pd
+from aggregate.aggregation_helpers import order_aggregated_columns
+from internal_review.set_internal_review_file import set_internal_review_files
+from utils.PUMA_helpers import borough_name_mapper, clean_PUMAs, get_all_boroughs
+from utils.dcp_population_excel_helpers import (
+    race_suffix_mapper_global,
+    count_suffix_mapper_global,
+    median_suffix_mapper_global,
+    remove_duplicate_cols,
+)
+
+occupations = ["mbsa", "srvc", "slsoff", "cstmnt", "prdtrn"]
```

Comment:
I agree that the mappers shouldn't have the underscores - I prefer your method of doing the mappers without underscores as it takes away from some of the confusion as to where in the mapper the underscore goes when it replaces the original

Reply:
on refactor, we can update the other scripts